test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/WorkItemTransactionTests.cs

```diff
@@ -15,12 +15,12 @@
 
 namespace Microsoft.Health.Dicom.Web.Tests.E2E.Rest;
 
-public partial class WorkItemTransactionTests : IClassFixture<HttpIntegrationTestFixture<Startup>>
+public partial class WorkItemTransactionTests : IClassFixture<DataPartitionEnabledHttpIntegrationTestFixture<Startup>>
```

Comment:
Can we fix the naming of the `DataPartitionEnabledHttpIntegrationTestFixture` and underlying parameter we pass from `DataPartitionEnabled...` to something like `FeaturesEnabled...` to be more explicit? It's kind of confusing to have to have `DataPartitions` to test `WorkItems` when the features are orthogonal.

Reply:
Done